docs/setup.md

```diff
@@ -114,6 +109,8 @@ After editing any schema, you have to create new database migrations to apply th
 The comment will go in the filename, so it should really be short.  
 Look out for any warnings, and review the created migration file before applying it (with `alembic upgrade head`).
 
+Note that in the devshell and test docker environments above, database migrations are automatically run when you create the environment.  In those cases, you will only need to run `alembic upgrade head` if you've created a new databse migration since you started up your docker environment.
```

Comment:
This is not clear: if you've run migrations then it sounds like you don't need to `upgrade head`. Or do you mean that if you left the same database service running and added a migration after it was already created, then you need to `upgrade head`?

Reply:
The latter.